code/extensions/QuickFeedbackExtension.php

```diff
@@ -26,6 +26,19 @@ class QuickFeedbackExtension extends DataExtension {
 	 * @return Form
 	 */
 	public function QuickFeedbackForm() {
+
+		// if HideFeedbackForm returns true then hide form
+		$page = $this->owner;
+		if($page->HideFeedbackForm) {
+			return false;
+		} else {
+			while($page->exists()) {
+				if($page->HideFeedbackForm) return false;
+
+				$page = $page->Parent();
```

Comment:
Can you explain the logic here? What happens if I have a parent page with the feedback form turned off, but I want a child page to still show?